HarryJLeigh.FlashCards/FlashCards/Controllers/FlashcardController.cs

```diff
@@ -0,0 +1,68 @@
+using System.Data.SqlClient;
+using Dapper;
+using FlashCards.Data;
+
+namespace FlashCards.Controllers;
+
+public class FlashcardController
+{
+    private DatabaseService DbService { get; }
+    
+    public FlashcardController(DatabaseService dbService) => DbService = dbService;
+    
+    public List<FlashCardDto> GetAllFlashcards(int stack_id) => ExecuteQuery("Select flashCardId, front, back FROM Flashcards WHERE stack_id = @Stack_id", new {Stack_id = stack_id });
```

Comment:
🟠 **Code Readability**

💡 If you are going to smash all this together, at least make it readable... Blank lines, line length limit 😄 please 😆

Reply:
Looking back at this, I realise the long line is quite hard to read. I've improved the readability by breaking it into multiple lines for better clarity. 😊